builtin/k8s/releaser.go

```diff
@@ -724,12 +724,18 @@ func (r *Releaser) Release(
 	ctx context.Context,
 	log hclog.Logger,
 	src *component.Source,
+	job *component.JobInfo,
 	ui terminal.UI,
 	target *Deployment,
 	dcr *component.DeclaredResourcesResp,
 ) (*Release, error) {
 	var result Release
-	result.ServiceName = src.App
+
+	if job.Workspace == "default" {
```

Comment:
I think we should throw this in the labels too for the label selector.

Reply:
Not sure what ya mean....